netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -258,6 +261,51 @@ _ebpf_sock_addr_get_current_logon_id(_In_ const bpf_sock_addr_t* ctx)
     return logon_id;
 }
 
+static int
```

Comment:
Return can be NTSTATUS.

Reply:
Return for helper functions should be int.